### PR TITLE
ENH: Add IOMeshSWC remote module.

### PR DESCRIPTION
**Pull request by thewtex**
Read meshes from SWC files, a format for representing neuron morphology.

### CODE REVIEW

**Review by tbirdso — APPROVED** [review]
LGTM

File: Modules/Remote/IOMeshSWC.remote.cmake [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#-- ## Compliance Level 0 star ( Code/Feature of known poor-quality or deprecated status )
+#--   - [ ] Code reviewed and explicitly identified as not recommended for use
+#--
+#-- ### Please document here any justification for the criteria above
+#       Code style enforced by clang-format on 2020-02-19, and clang-tidy modernizations completed
+
+# Contact: Matt McCormick <matt.mccormick@kitware.com>
+itk_fetch_module(IOMeshSWC
+  "Read meshes from SWC files, a format for representing neuron morphology."
+  MODULE_COMPLIANCE_LEVEL 3
```
Comment (dzenanz) [inline Modules/Remote/IOMeshSWC.remote.cmake:47]: Judging by the evaluation above, this should be 4.

File: Modules/Remote/IOMeshSWC.remote.cmake [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#-- # Grading Level Criteria Report
+#-- EVALUATION DATE: 2022-03-08
+#-- EVALUATORS: [Matt McCormick
```
Comment (dzenanz) [inline Modules/Remote/IOMeshSWC.remote.cmake:3]: missing `]`

File: Modules/Remote/IOMeshSWC.remote.cmake (hunk 1 above)
Comment (thewtex) [reply Modules/Remote/IOMeshSWC.remote.cmake:47]: :+1:

File: Modules/Remote/IOMeshSWC.remote.cmake (hunk 2 above)
Comment (thewtex) [reply Modules/Remote/IOMeshSWC.remote.cmake:3]: :+1:

**Pull request merged**